src/antsibull/cli/antsibull_build.py

```diff
@@ -131,6 +131,16 @@ def _normalize_new_release_options(args: argparse.Namespace) -> None:
             )
 
 
+def _check_release_build_directories(args: argparse.Namespace) -> None:
+    if args.command in ('single', 'multiple', 'rebuild-single'):
+        if not os.path.isdir(args.sdist_dir):
+            raise InvalidArgumentError(f'{args.sdist_dir} must be an existing directory')
+
+    if args.command in ('rebuild-single', ):
```

Comment:
This seems like a convoluted way to check the equality of two strings.  

```suggestion
    if args.command == 'rebuild-single':
```

Reply:
I prefer all `args.command` comparisons in this function to work the same way.